tools/patches.c

```diff
@@ -152,9 +152,43 @@ build_process_list(void)
   return list;
 }
 
-/* Prints all the info collected about the processes in PROCESS_LIST. */
+/** @brief Convert build id provided in `build_id` into string.
+ *
+ * String is returned as having spaces between bytes. Example:
+ *
+ * with buildid: 338aa4d16c98dda7af170cc8e2b59d259bd5d4f4
+ *
+ * it will return the string:
+ * "33 8a a4 d1 6c 98 dd a7 af 17 0c c8 e2 b5 9d 25 9b d5 d4 f4"
```

Comment:
Please make the string not have spaces, it should be a single "identifier" (which happens to be a long hex number).